apps/charterafrica/src/pages/Explainers.page.js

```diff
@@ -0,0 +1,31 @@
+import Explainers from "../components/Explainers";
```

Comment:
Haven't checked this out much but `explainers` should be a _child_ of `knowledge` page... We'll discuss them after getting the other parts working perfectly.